ci-operator/step-registry/cucushift/upgrade/toimage/cucushift-upgrade-toimage-commands.sh

```diff
@@ -238,6 +238,14 @@ function cco_annotation(){
     fi
 }
 
+function disable_boot_image_update() {
+    echo "Disabling updated boot images by editing MachineConfiguration..."
```

Comment:
Suggest to add a check if MachineConfiguration/cluster is already patched or not, because cucushift-upgrade-toimage will be run as the last hop upgrade step.

Reply:
updated